data_store/controllers/ingest.py

```diff
@@ -518,3 +520,14 @@ def save_failed_submission(file: IO) -> uuid.UUID:
     s3_object_name = FAILED_FILE_S3_NAME_FORMAT.format(failure_uuid, datetime.now().strftime(DATETIME_ISO_8601))
     upload_file(file=file, bucket=Config.AWS_S3_BUCKET_FAILED_FILES, object_name=s3_object_name)
     return failure_uuid
+
+
+def make_ascii_safe_filename(filename: str) -> str:
+    """
+    Converts a filename to an ASCII-safe format.
+    """
+    base, ext = os.path.splitext(filename)
+
+    base = "".join((c if ord(c) < 128 else "-") for c in base)
+
+    return f"{base}{ext}"
```

Comment:
Could we get a set of explicit unit tests for this that make it clear what goes in and what comes out? It feels like a nice self-contained thing to test.

Reply:
Sure!